### PR TITLE
Archeny my room and comp

### PR DESCRIPTION
**Pull request by contingency40**
### Completed

* Archeny chat dialogue
* Archeny bonding dialogue
* Archeny engagement dialogue
* Archeny compendium entries
* Archeny gift dialogue
* Archeny battle advice
* Aconitum compendium entires
* Archeny and Aconitum milking scenes

### CODE REVIEW

File: Gaijinizer patch/Enemies.txt [hunk 1]
```diff
@@ -1904,7 +1904,7 @@ It would have been wanted to be "loved" existence.#MTLed
 > BEGIN STRING
 氷以外に少し強い
 > CONTEXT: Enemies/574/BookAttribute/2
-A little stronger than ice#MTLed
+Slightly Resists All except Ice#RonIffyWording
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Enemies.txt:1908]: "except" -> "but"

File: Gaijinizer patch/Troops.txt [hunk 2]
```diff
@@ -7975,50 +7975,57 @@ Archeny 【Complete book】 Embrace#MTLed
...
 > BEGIN STRING
 おや？いやだったかい？
 
 てっきりお前さんは、強引なのが好きだと思ってたよ。
 > CONTEXT: Troops/185/pages/0/39/Dialogue/enemy10
-Oh? Are you sorry? I thought that you liked to be
-obsessive.#MTLed
+Oh? You don't like it? I was under the impression
+you enjoyed being forced.#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Troops.txt:7990]: Note

File: Gaijinizer patch/Troops.txt [hunk 3]
```diff
@@ -7975,50 +7975,57 @@ Archeny 【Complete book】 Embrace#MTLed
...
 > BEGIN STRING
 へぇ……流石に本気とはいかなくとも
 
 合意の上だったとしても抵抗するのかい…
 > CONTEXT: Troops/185/pages/0/55/Dialogue/enemy10
-Hee...without being serious about it
-Do you struggle if you agree?#MTLed
+Hmm...? So you are struggling even though you
+agreed to this?#RonIffy
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Troops.txt:8019]: Note

File: Gaijinizer patch/Enemies.txt [hunk 4]
```diff
@@ -1953,11 +1953,11 @@ You don't have to rush to chase your prey.#MTLed
...
-Her eyes are the red four parts of the head.
-The lower jaw has only a pseudo-mouth that speaks and gives
-a tongue
-All other organs are at other sites and do not exist there.#MTLed
+The human-like part of her face is actually her
+entire lower jaw. Her eyes are the four red parts
+on her head. That lower jaw is nothing more than
+a pseudo-mouth with a tongue used for talking,
+it has no other organs built into it.#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Enemies.txt:1961]: Minor flow:
"That" -> "The"

File: Gaijinizer patch/Enemies.txt [hunk 5]
```diff
@@ -2014,13 +2012,12 @@ Physical avoidance ability is high#MTLed
...
-The prey which has succumbed is put into the body from the
-abdomen
-You can liquefy your body as a preservation diet for
-Archeny.#MTLed
+A familiar that Archeny released into her nest.
+It affixes itself to its preys crotch, and starts
+milking them with its tongue. Prey that's
+succumbed to Aconitum soon find their body
+engulfed up to their waist, and their lower half
+liquefied.#Ron
```
Comment (Triketos) [inline Gaijinizer patch/Enemies.txt:2020]: "that's" -> "that have"

File: Gaijinizer patch/Enemies.txt [hunk 6]
```diff
@@ -1933,12 +1933,12 @@ before you ask.#MTLed
...
-She walks up and down alternately like a human being,
-standing upright.
-You don't have to rush to chase your prey.#MTLed
+Whether capturing prey or merely walking, she
+uses her legs much like a normal spider, and can
+move quite nimbly. But when standing upright she
+walks around like a human does. This is because
+there's no need for her to rush when chasing her
+prey.#RonIffy
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Enemies.txt:1942]: Note for Ron to double check.

File: Gaijinizer patch/Enemies.txt [hunk 7]
```diff
@@ -2036,13 +2033,12 @@ Archeny.#MTLed
...
-Because there is no milking tongue and there is no magic
-power of familiarity itself
-Although ordinary people can not stand it, it will not be
-as good as her.#MTLed
+This familiar has multiple tongues, using them to
+dexterously entangle around their prey, and
+perform exquisite fellatio. They're similiar in
+color and composition to Empuse's, though there's
+no magic infused in them. So while formidable to
+a human, it doesn't compare to Empuse at all.#RonIffyWording
```
Comment (Triketos) [inline Gaijinizer patch/Enemies.txt:2041]: A few minor bits of cleanup here:
>_**This familiar has multiple tongues, using them to
dexterously entangle around their prey and
perform exquisite fellatio. They're similiar in
color and composition to Empuse's, but there's no
magic infused in them. So while formidable against
a human, they don't compare to Empuse's at all.**_

File: Gaijinizer patch/Map119.txt [hunk 8]
```diff
@@ -233,28 +239,31 @@ It's really strange. How about you?#MTLed
...
 あるクソ女神のせいで、こんな悪魔に転生しちまったがね。
 > CONTEXT: Map119/events/5/pages/1/51/Dialogue/enemy10
-I was originally a woman who weaved
-Because of a certain fucking goddess, I was
-incarnated to such a demon.#MTLed
+I used to be a weaver-woman...
+
+But because of a certain bitch goddess, I was
+reincarnated in this form.#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map119.txt:267]: As we mentioned before, it's likely referencing this: https://en.wikipedia.org/wiki/Arachne

I'll be suggesting a few changes to align the two together.

File: Gaijinizer patch/Map119.txt [hunk 9]
```diff
@@ -263,9 +272,10 @@ incarnated to such a demon.#MTLed
...
  あっちゃいけないからねぇ。
 > CONTEXT: Map119/events/5/pages/1/55/Dialogue/enemy10
-...... This is a long time ago story.
-The goddess who controls the fabric can defeat an
-ordinary human and lose. I can't wait.#MTLed
+Kuku... This is a story from a long time ago.
+
+The story of the goddess of textiles being
+defeated by a mere human.#RonIffy
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map119.txt:279]: Is this a valid interpretation, keeping in with text?

>_**The story of a goddess being defeated by
the weaving of a mere human.**_

File: Gaijinizer patch/Map119.txt [hunk 10]
```diff
@@ -274,47 +284,52 @@ ordinary human and lose. I can't wait.#MTLed
...
 > CONTEXT: Map119/events/5/pages/1/63/Dialogue/enemy10
-Oops! I will nail the nails before I make a
-strange mistake. I wanted to change my
-personality since I was human. It was from the
-person who wanted to wear a crab first.#MTLed
+Whoops! Let me warn you before you start getting
+any funny ideas: My personality was twisted even
+while I was human. I was the one who picked a
+fight with her.#RonIffy
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map119.txt:291]: Note

File: Gaijinizer patch/Map119.txt [hunk 11]
```diff
@@ -274,47 +284,52 @@ ordinary human and lose. I can't wait.#MTLed
...
 今でも、自分の方が織物の腕は上だったと思ってるよ。
 
 だけどあの時のあたいは、少し愚かだったのさ。
 > CONTEXT: Map119/events/5/pages/1/68/Dialogue/enemy10
-Even now, I think that I was the upper arm of the
-fabric. But that time, I was a little stupid.#MTLed
+Even now, I still think my textiles were better.
+
+But I was a little foolish back then.#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map119.txt:301]: "textiles" -> "weaving"

File: Gaijinizer patch/Map119.txt [hunk 12]
```diff
@@ -274,47 +284,52 @@ ordinary human and lose. I can't wait.#MTLed
...
 
 あのクソ女神だって、最初は忠告しに来ていたんだよ。
 > CONTEXT: Map119/events/5/pages/1/72/Dialogue/enemy10
-Admired from around, self-indulgent ......
-Even that fucking goddess was coming for advice
-at first.#MTLed
+Praised by everyone, full of conceit...
+
+Even that bitch goddess was coming to me for
+advice...#Ron
```
Comment (Triketos) [inline Gaijinizer patch/Map119.txt:311]: Let's alternate the wording here: "bitch" -> "shitty"

File: Gaijinizer patch/Map119.txt [hunk 13]
```diff
@@ -274,47 +284,52 @@ ordinary human and lose. I can't wait.#MTLed
...
 だからこそ、転生させられたこの身体は
 
 あたいを『戒める』要素がたっぷり詰まっている。
 > CONTEXT: Map119/events/5/pages/1/76/Dialogue/enemy10
-That's why this reincarnate body is
-There is a lot of elements "Keep it up" for you.#MTLed
+That's why I was reincarnated in this body...
+
+All part of my "punishment".#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map119.txt:322]: Minor text:
>_**All as part of my "punishment".**_

File: Gaijinizer patch/Map119.txt [hunk 14]
```diff
@@ -323,28 +338,31 @@ Like a veil, you're all in one and beautiful?#MTLed
...
  一見、便利な身体の一部さ。
 > CONTEXT: Map119/events/5/pages/1/85/Dialogue/enemy10
-If you wrap it up, it will be assimilated to the
-scenery around you. There is also a camouflage
-function, you can freely put in and out of your
-head. A seemingly convenient part of the body.#MTLed
+Clad in this hair, I seamlessly blend into
+the background. It also has camouflage abilities
+and I can draw and retract it as needed. At first
+glance, it's quite a conveniant part of my body.#RonLocalization
```
Comment (Triketos) [inline Gaijinizer patch/Map119.txt:344]: "conveniant" -> "convenient"

File: Gaijinizer patch/Map119.txt [hunk 15]
```diff
@@ -365,7 +383,7 @@ goddess, "Weaving woven fabrics" as it is.#MTLed
 > CONTEXT: Map119/events/5/pages/1/105/Dialogue/enemy10
 That fucking goddess is not willing to forgive.
 I've woven such a thing with a wicked heart, I
-wish you the love of that day.#MTLed
+wish you the love of that day.#CheckLater
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map119.txt:387]: You'll need to recheck this...

File: Gaijinizer patch/Map119.txt [hunk 16]
```diff
@@ -568,89 +596,94 @@ Why so many milking parts!#MTLed
...
 
 あのクソ女神も、いい趣味してるね、まったく……
 > CONTEXT: Map119/events/5/pages/1/239/Dialogue/enemy10
-...... although I can not say the other party
-That fucking goddess also has a good hobbies,
-absolutely ...#MTLed
+Though I can't say much about her personality,
+that bitch goddess sure has some twisted
+hobbies...#RonSUPERIffy
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map119.txt:677]: Note

File: Gaijinizer patch/Map119.txt [hunk 17]
```diff
@@ -568,89 +596,94 @@ Why so many milking parts!#MTLed
...
 じゃなきゃ、かつての人の身体の部位を残して
 
 呪いであたいを転生させたりしないだろうよ。
 > CONTEXT: Map119/events/5/pages/1/235/Dialogue/enemy10
-If not, leave the body part of the former person
-The curse will not incarnate you.#MTLed
+If I don't, and hold on to the remaining parts
+of my former human body, the curse will likely
+not allow me to reincarnate.#RonIffy
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map119.txt:667]: Note

File: Gaijinizer patch/Map119.txt [hunk 18]
```diff
@@ -1349,9 +1433,10 @@ demon's attention.#MTLed
...
 そこまでは自然な流れさ。
 > CONTEXT: Map119/events/7/pages/0/59/Dialogue/enemy10
-You ran away from the nest to survive.
-I was caught up with you to meet him.
-It is a natural flow up to that point.#MTLed
+You escaped my nest in order to survive.
+I stood in your way in order to eat you.
+That was the natural flow of things up to that
+point.#MTLed
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map119.txt:1440]: Might want to add credit here :3

File: Gaijinizer patch/Map119.txt [hunk 19]
```diff
@@ -1311,36 +1390,41 @@ taste is exceptional, so it's the best...♡#MTLed
...
 > BEGIN STRING
 本来は他人の前で見栄をはったり意地をはったりするヤツが
 
 あたいに屈服する姿を見るのは、本当に愉快だからねぇ…♡
 > CONTEXT: Map119/events/7/pages/0/44/Dialogue/enemy10
-The guy who looks and loves in front of others
-It's really fun to see him giving in on his face.#MTLed
+It's so much more amusing to see someone break
+who's stubborn, someone who puts on airs in front
+of others.#RonIffyWording
```
Comment (Triketos) [inline Gaijinizer patch/Map119.txt:1406]: A bit of rewording:
>_**It's so much more amusing to break someone who's
stubborn, and who puts on airs in front of others...♡**_

File: Gaijinizer patch/Map119.txt [hunk 20]
```diff
@@ -440,93 +466,96 @@ with love juice.#MTLed
...
 細かい振動を繰り返して、相手をくすぐり続ける。
 > CONTEXT: Map119/events/5/pages/1/151/Dialogue/enemy10
-So the yarn entwined with the prey stretches at
-the outside temperature. Repeat the fine
-vibration and keep tickling the other party.#MTLed
+The thread tangling around my prey expand and
+contracts through external temperature, and
+repeatedly makes fine movements, continuously
+tickling my opponent.#RonIffyWordingWhatTheHellIsThisJapanese
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map119.txt:484]: Note

File: Gaijinizer patch/Map119.txt (hunk 9 above)
Comment (contingency40) [reply Gaijinizer patch/Map119.txt]: That works well.

File: Gaijinizer patch/Troops.txt [hunk 21]
```diff
@@ -8035,10 +8044,10 @@ Archeny 【Whole book】 Pin Down#MTLed
...
 > CONTEXT: Troops/208/pages/0/80/Dialogue/enemy10
-Usually I usually leave it to Aconitum
-This way you don't hold your own game slowly.
-Mostly, it will be the standing milking after
-pulling#MTLed
+I usually leave this kind of thing to Aconitum,
+so I don't often get to leisurely hold my prey
+like this. This is normally a position I get in
+after reeling in my prey.#RonIffy
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Troops.txt:8051]: Minor flow:
>_**This is normally a position I get in
after reeling them in...**_

**Pull request merged**